Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -2685,7 +2685,7 @@ void main()
                     Assert.IsTrue(attached.WaitOne(20000), "Failed to attach within 20s");
                     Assert.IsTrue(bpHit.WaitOne(20000), "Failed to hit breakpoint within 20s");
 
-                    p.WaitForExit(10000);
+                    p.WaitForExit(DefaultWaitForExitTimeout);

```

Comment:
It won't work because it expects a `PythonProcess`. And the reason why we can't wait on `proc` in this snippet is because it doesn't work for remote processes - you'll get `InvalidOperationException` if you do. This, in turn, is because `PythonProcess.WaitForExit` really just delegates to `Process.WaitForExit` of the process that it wraps; and there's no `Process` instance for remote processes.